examples/Integration Mappings/Integration-mappings.ps1

```diff
@@ -0,0 +1,265 @@
+# This script can do one of the following things: 
+# 1. Create a csv of all the current mappings between Box folders and Slack channels
+# 2. Create a mapping between a Box folder and a Slack channel based on a list in a csv file
+
+#APP SETUP
+#Oauth 2.0: User configured with CLI must be admin or co-admin
+#APPLICATION SCOPES: Read all files and folders stored in Box, & Manage Enterprise Settings
+
+#############################################################################
+
+param (
+    [string]$action
+)
```

Comment:
How about adding a new param with path to `mapping.csv` and use this param instead of hardcoded value?
We can even set this to default value to `mappings.csv` by sth like that:

`[string]$MappingPath = "./mapping.csv",`

This way the user will be able to pass mapping from any path.